user-manual/index.rst

```diff
@@ -6,10 +6,12 @@
 CTI |release|
 =============
 
-Generale
---------
+.. sidebar:: Vedi anche
+
+  * `NethCti 2  <https://nethcti.docs.nethesis.it/it/v2/index.html>`_
```

Comment:
Please change this label to `CTI 2`, so it will fit the page title

Reply:
done